### PR TITLE
Move openapi spec, update list repositories

### PR DESCRIPTION
**Pull request by mshriver**
Trailing slash is required by the routing, but wasn't included in the spec.  Auto-generated openapi client (IQE) ends up building an invalid URL.

Move from docs to api directory, repository boilerplate indicates that's where it should go.

I've built and run the container locally to validate the repositories endpoint still works for GET. 
I've taken the generated spec and fed it into the IQE openapi client generator to validate the client using the correct URL. 
Trying to deploy that container within ephemeral and make an IQE rest client request against it now.

### CODE REVIEW

File: pkg/handler/api.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"path/filepath"
 	"strconv"
 
-	"github.com/content-services/content-sources-backend/docs"
+	spec_api "github.com/content-services/content-sources-backend/api"
```
Comment (mshriver) [inline pkg/handler/api.go:13]: Would love input on this naming @jlsherrill

**Comment by mshriver** [comment]
The `Format` check is failing on one of the auto-generated files, could use some advice there.

File: pkg/handler/api.go (hunk 1 above)
Comment (jlsherrill) [reply pkg/handler/api.go:13]: that looks fine to me!

File: api/openapi.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //Used for embedding & fetching the openapi doc at build time, so it can be retrieved
-package docs
+package api
```
[the remark this answers is not in the document]
Comment (jlsherrill) [reply api/openapi.go]: yes, lets do that!

**Review by jlsherrill — APPROVED** [review]

File: api/openapi.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (jlsherrill) [reply api/openapi.go]: for most of us using an ide, it happens automatically so we don't think about it.

**Pull request merged**